doc/rtmpwebhookauth.md

```diff
@@ -39,4 +40,6 @@ An optional streamKey may be provided in order to protect the RTMP stream from p
 
 Presets can be specified to override the default transcoding options. The available presets are listed [here](https://github.com/livepeer/go-livepeer/blob/master/common/videoprofile_ids.go).
 
+Custom transcoding profiles can be provided if the presets are not sufficient. Given a stream name (manifest ID) of "ManifestID" and a profile name of "ProfileName", the specific profile will be available for playback at `/stream/ManifestID/ProfileName.m3u8`. However, to take advantage of ABR features in HLS players, the top-level stream name should usually be supplied instead, eg `/stream/ManifestID.m3u8` The `bitrate` field is in bits per second. The `fps` field can be omitted to preserve the source frame rate.
```

Comment:
I would add, after `Custom transcoding profiles can be provided if the presets are not sufficient` ... `Both presets and profiles can be used together to specify desired transcodes.` The rest looks good!

Reply:
Fixed in a0bac79c